client/pkg/fileutil/fileutil.go

```diff
@@ -135,3 +134,35 @@ func CheckDirPermission(dir string, perm os.FileMode) error {
 	}
 	return nil
 }
+
+// RemoveMatchFile deletes file if matchFunc is true on an existing dir
+// Returns error if the dir does not exist or remove file fail
+func RemoveMatchFile(lg *zap.Logger, dir string, matchFunc func(fileName string) bool) error {
+	if lg == nil {
+		lg = zap.NewNop()
+	}
+	if !Exist(dir) {
+		return fmt.Errorf("directory %q does not exist", dir)
```

Comment:
What's %q here: 

`a single-quoted character literal safely escaped with Go syntax. ` ?

Should this be '%s' ?